core/cmd/nodes_commands.go

```diff
@@ -65,4 +65,4 @@ func (cli *nodeClient[P2]) IndexNodes(c *cli.Context) (err error) {
 	return cli.getPage(cli.path, c.Int("page"), &p)
 }
 
-var nodeHeaders = []string{"Name", "Chain ID", "State", "Config"}
+var nodeHeaders = []string{"ID", "Name", "Chain ID", "State", "Config"}
```

Comment:
What is `ID`? IIRC that was excluded by design, since we no longer have database IDs.

Reply:
I thought that ID/JAID was somehow important, in that case, it's better to just remove it.

Reply:
It may be important for graphQL? but we don't need to display it for users since we just re-use 'name' now.